homeassistant/components/co2signal/coordinator.py

```diff
@@ -3,6 +3,7 @@
 
 from collections.abc import Mapping
 from datetime import timedelta
+from json import JSONDecodeError
```

Comment:
@joostlek @jpbede 

Looking at the exception thrown in the issue linked:

```
requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

However, it seems not the exception imported and catched here?
The tests in this PR mocks with the exception (instead of causing the exception), so it is bogus for this specific error testing.

Reply:
#101206 :)

Reply:
🤦 ok...

Reply:
Although now you mention it, we did not change the test for this one